bin/deploy.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# This file is used to deploy addons-blog to an S3 bucket.
+# You need to set the S3 bucket name in an environment variable named
+# ADDONS_BLOG_BUCKET.
+
+set -ex
+
+if [ ! -d "dist" ]; then
+    echo "Can't find /dist/ directory. Are you running from the correct"\
+         "root directory?"
+    exit 1
+fi
+
+if [ -z "$ADDONS_BLOG_BUCKET" ]; then
+    echo "The S3 bucket is not set. Failing."
+    exit 1
+fi
+
+# The basic strategy is to sync all the files that need special attention
+# first, and then sync everything else which will get defaults
+
+
+# For short-lived assets; in seconds
+TEN_MINUTES="600"
+
+# For long-lived assets; in seconds
+ONE_YEAR="31536000"
+
+CSPSTATIC="\"content-security-policy\": \"default-src 'none'; "\
+"base-uri 'none'; "\
+"form-action 'none'; "\
+"object-src 'none'\""
+
+# DO NOT REMOVE the locale from the newsletter form url. See #476.
+CSP="\"content-security-policy\": \"default-src 'none'; "\
+"base-uri 'self'; "\
+"connect-src 'self' https://blog.mozilla.org/addons/feed/ https://www.mozilla.org/en-US/newsletter/ https://www.google-analytics.com/; "\
+"font-src 'self'; "\
+"form-action 'self' https://www.mozilla.org/en-US/newsletter/; "\
+"frame-ancestors 'none'; "\
+"frame-src https://www.youtube.com/embed/; "\
+"img-src 'self' data:; "\
+"object-src 'none'; "\
+"script-src 'self' https://www.youtube.com/iframe_api https://www.youtube.com/s/player/ 'sha256-cEol3PeVsUqXLYFr6jKFJdNafMQx9RDvCSi6+kCHa6U=' https://www.google-analytics.com/analytics.js; "\
+"style-src 'self' 'unsafe-inline'\""
```

Comment:
```suggestion
# DO NOT REMOVE the locale from the newsletter form url. See #476.
CSP="\"content-security-policy\": \"default-src 'none'; "\
"base-uri 'self'; "\
"connect-src 'self' https://www.google-analytics.com/; "\
"font-src 'self'; "\
"img-src 'self'; "\
"object-src 'none'; "\
"script-src 'self' https://www.google-analytics.com/analytics.js; "\
"style-src 'self' 'unsafe-inline'\""
```

Reply:
@muffinresearch I removed some of the entries because the default (`none`) should be applied. Is that OK? Also, could you double check? I think we don't need a lot of config, but I kept GA for #85

Reply:
@willdurand: You'll want to keep `form-action` and `frame-ancestors`.

`base-uri`, `form-action`, `frame-ancestors` don't fall-back to `default-src`.

Reply:
aww thanks

Reply:
(I updated the suggestion)

Reply:
FWIW (and this can be tweaked later) Looks like the demo content has youtube, though wp seems to handle embeds with JS too. https://mozamo.wpengine.com/2021/04/12/all-content-test/